terraform/autoscaling.tf

```diff
@@ -95,98 +95,98 @@ resource "aws_cloudwatch_metric_alarm" "wfprev_service_cpu_low" {
+#      }
+#    }
+
+#    depends_on = [aws_appautoscaling_target.wfprev_client_target]
+#  }
 
 # CloudWatch alarm that triggers the autoscaling down policy
-resource "aws_cloudwatch_metric_alarm" "wfprev_client_service_cpu_low" {
-   alarm_name          = "wfprev_client_cpu_utilization_low"
-   comparison_operator = "LessThanOrEqualToThreshold"
-   evaluation_periods  = "2"
-   metric_name         = "CPUUtilization"
-   namespace           = "AWS/ECS"
-   period              = "60"
-   statistic           = "Average"
-   threshold           = "10"
-
-   dimensions = {
-     ClusterName = aws_ecs_cluster.wfprev_main.name
-     ServiceName = aws_ecs_service.client.name
-   }
-
-   alarm_actions = [aws_appautoscaling_policy.wfprev_client_down.arn]
-
-   tags = {
-     Environment = "${var.TARGET_ENV}"
-   }
- }
+# resource "aws_cloudwatch_metric_alarm" "wfprev_client_service_cpu_low" {
+#    alarm_name          = "wfprev_client_cpu_utilization_low"
+#    comparison_operator = "LessThanOrEqualToThreshold"
+#    evaluation_periods  = "2"
+#    metric_name         = "CPUUtilization"
+#    namespace           = "AWS/ECS"
+#    period              = "60"
+#    statistic           = "Average"
+#    threshold           = "10"
+
+#    dimensions = {
+#      ClusterName = aws_ecs_cluster.wfprev_main.name
+#      ServiceName = aws_ecs_service.client.name
+#    }
+
+#    alarm_actions = [aws_appautoscaling_policy.wfprev_client_down.arn]
+
+#    tags = {
+#      Environment = "${var.TARGET_ENV}"
+#    }
+#  }
 
 #  CloudWatch alarm that triggers the autoscaling up policy
-resource "aws_cloudwatch_metric_alarm" "wfprev_client_service_cpu_high" {
-   alarm_name          = "wfprev_client_cpu_utilization_high"
-   comparison_operator = "GreaterThanOrEqualToThreshold"
-   evaluation_periods  = "1"
-   metric_name         = "CPUUtilization"
-   namespace           = "AWS/ECS"
-   period              = "60"
-   statistic           = "Average"
-   threshold           = "50"
-
-   dimensions = {
-     ClusterName = aws_ecs_cluster.wfprev_main.name
-     ServiceName = aws_ecs_service.client.name
-   }
-
-   alarm_actions = [aws_appautoscaling_policy.wfprev_client_up.arn]
-
-   tags = {
-     Environment = "${var.TARGET_ENV}"
-   }
- }
\ No newline at end of file
+# resource "aws_cloudwatch_metric_alarm" "wfprev_client_service_cpu_high" {
+#    alarm_name          = "wfprev_client_cpu_utilization_high"
+#    comparison_operator = "GreaterThanOrEqualToThreshold"
+#    evaluation_periods  = "1"
+#    metric_name         = "CPUUtilization"
+#    namespace           = "AWS/ECS"
+#    period              = "60"
+#    statistic           = "Average"
+#    threshold           = "50"
+
+#    dimensions = {
+#      ClusterName = aws_ecs_cluster.wfprev_main.name
+#      ServiceName = aws_ecs_service.client.name
+#    }
+
+#    alarm_actions = [aws_appautoscaling_policy.wfprev_client_up.arn]
+
+#    tags = {
+#      Environment = "${var.TARGET_ENV}"
+#    }
+#  }
```

Comment:
I take it load balancing and autoscaling is not needed now that the UI is handled in CloudFront and S3? Are they something that comes with CloudFront by default or do we need to do more work for that?